docs/Guide.Jest.md

```diff
@@ -46,13 +46,15 @@ Add this part to your `package.json`:
   "setupTestFrameworkScriptFile": "./e2e/init.js"
 },
 "scripts": {
-    "test:e2e": "detox test",
+    "test:e2e": "detox test -c ios.sim.debug -a e2e",
```

Comment:
-a sets artifacts output location, what did you mean by that?

Reply:
for some reason, if you don't specify anything, `_getAbsolutePath` tries to create artifacts folder under `undefined/`

Reply:
That’s a different issue then, but we certainly don’t want them in our e2e tests library (or anywhere for that matter, it’s unrelated to this part of the docs). Let’s open an issue to fix that ASAP

Reply:
I think it is related to `process.cwd()` in child process. I'll follow up on this

Reply:
@rotemmiz ok, it seems to happen only if I use `xcode` config (to debug using xcode), because `binaryPath` is not specified. Should I make an exception here?

Reply:
Not sure I follow why `_getAbsolutePath` is needed in that case, but I'll take a look in the code later.
I still think the issue you're trying to debug is out of the scope of  this PR, so we can change the docs, and open a new issue with that.

Reply:
Sorry for miscommunication. `process.cwd()` has nothing to do with the issue. Yeah, I can remove artifacts part as far as it is only related to `xcode` debug configuration. I'll follow up on this in separated issue